tests/common/devices.py

```diff
@@ -1065,6 +1066,83 @@ def get_extended_minigraph_facts(self, tbinfo):
         return mg_facts
 
 
+class K8sMasterHost(AnsibleHostBase):
+    """
+    @summary: Class for Ubuntu KVM that hosts Kubernetes master
+
+    For running ansible module on the K8s Ubuntu KVM host
+    """
+
+    def __init__(self, ansible_adhoc, hostname, ip_addr, k8s_user, k8s_passwd):
+        '''Initialize an object for interacting with Ubuntu KVM using ansible modules
+        
+        Args:
+            ansible_adhoc (): The pytest-ansible fixture
+            hostname (string): hostname of the Ubuntu KVM
+            k8s_user (string): Username for accessing the K8s VM CLI interface
+            k8s_passwd (string): Password for the k8s_user 
+        '''
+        self.k8s_user = k8s_user
+        self.k8s_passwd = k8s_passwd
+        self.hostname = hostname
+        self.ip_addr = ip_addr
+        super(K8sMasterHost, self).__init__(ansible_adhoc, hostname)
+        evars = {
+            'ansible_user': self.k8s_user,
+            'ansible_ssh_user': self.k8s_user,
+            'ansible_ssh_pass': self.k8s_passwd,
+            'ansible_become_method': 'enable'
```

Comment:
The `ansible_ssh_user` and `ansible_ssh_pass` have already specified in group vars `ansible/group_vars/k8s_ubu/creds.yml`. It's unnecessary to specify them here again. Then the argument signature of `K8sMasterHost.__init__` can be more concise.

Reply:
I got some strange error when I tried to remove these from the argument signature of `K8sMasterHost;` it didn't seem to connect to the right host. I will look into it

Reply:
Thanks, I removed the unnecessary credential arguments.